lib/core/src/lib/comments/comments.component.spec.ts

```diff
@@ -122,6 +122,23 @@ describe('CommentsComponent', () => {
         expect(testingUtils.getByCSS('#comment-input')).not.toBeNull();
     });
 
+    it('should emit commentAdded when a new comment is added successfully', () => {
+        const emitSpy = spyOn(component.commentAdded, 'emit');
+        const mockComment = {
+            id: 'comment-123',
+            message: 'New test comment',
+            created: new Date().toISOString()
```

Comment:
toISOString returns string but if you check proper type for created property in CommentModel then it should be Date type instead of string